components/hero-unit.jsx

```diff
@@ -7,11 +7,16 @@ var HeroUnit = React.createClass({
   render: function() {
     return (
       <div className="row">
-        <div className="col-md-12 hero-unit" style={{
+        <div className="col-md-12 hero-unit hero-unit-1x" style={{
           backgroundImage: 'url(' + this.props.image + ')'
         }}>
           {this.props.children}
         </div>
+        <div className="col-md-12 hero-unit hero-unit-2x" style={{
+          backgroundImage: 'url(' + this.props.image2x || this.props.image + ')'
```

Comment:
this is a fantastic way to deal with the fallback if someone doesn't include the 2x in the `HeroUnit` tag .